diff --git a/src/define.h b/src/define.h
index 1b9b44b..4df8af9 100644
--- a/src/define.h
+++ b/src/define.h
@@ -54,7 +54,7 @@
 #define END_DET_TYPE                (0)
 
 /*Display AI frame rate*/
-#define DISP_AI_FRAME_RATE
+#define DISP_AI_FRAME_RATE  (0)
 
 /* Padding input mode to maintain the aspect ratio of DRP-AI input image.
    This mode requires the DRP-AI object file having the squared input size CAM_IMAGE_WIDTH x CAM_IMAGE_WIDTH.
diff --git a/src/main.cpp b/src/main.cpp
index ac1d576..20ceff8 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -60,7 +60,7 @@ MeraDrpRuntimeWrapper runtime;
 PreRuntime preruntime;
 
 static double drpai_time = 0;
-#ifdef DISP_AI_FRAME_RATE
+#if (1) == DISP_AI_FRAME_RATE
 static double ai_fps = 0;
 static double cap_fps = 0;
 static double proc_time_capture = 0;
@@ -367,12 +367,12 @@ void DRPAINode::R_Post_Proc(float* floatarr)
     filter_boxes_nms(det_buff, det_buff.size(), TH_NMS);
 
     /* Convert YUYV image to BGRA format.(bounding box algorithm require BGRA format) */
-    img.convert_format();
+    //img.convert_format();
 
     /* Draw bounding box on image. */
-    draw_bounding_box();
+    //draw_bounding_box();
 
-    cv::Mat out_image(CAM_IMAGE_HEIGHT, CAM_IMAGE_WIDTH, CV_8UC4, img.img_buffer[0]);
+    //cv::Mat out_image(CAM_IMAGE_HEIGHT, CAM_IMAGE_WIDTH, CV_8UC4, img.img_buffer[0]);
 
     /* Publish bounding box */
     darknet_ros_msgs::msg::BoundingBoxes bounding_boxes_results;
@@ -411,14 +411,14 @@ void DRPAINode::R_Post_Proc(float* floatarr)
     object_publisher->publish(msg);
 
     /* Publish detection image */
-    cv_bridge::CvImage cv_image;
-    cv_image.header.stamp = this->now();
-    cv_image.header.frame_id = "detection_image";
-    cv_image.encoding = "bgra8";
-    cv_image.image = out_image;
-    sensor_msgs::msg::Image ros_img;
-    cv_image.toImageMsg(ros_img);
-    detection_image_publisher->publish(ros_img);
+    //cv_bridge::CvImage cv_image;
+    //cv_image.header.stamp = this->now();
+    //cv_image.header.frame_id = "detection_image";
+    //cv_image.encoding = "bgra8";
+    //cv_image.image = out_image;
+    //sensor_msgs::msg::Image ros_img;
+    //cv_image.toImageMsg(ros_img);
+    //detection_image_publisher->publish(ros_img);
 
     mtx.lock();
     /* Clear the detected result list */
@@ -597,7 +597,7 @@ void *DRPAINode::R_Inf_Thread()
         drp_prev_time = inf_end_time;
         double total_time = ai_time + pre_time + post_time;
 
-#ifdef DISP_AI_FRAME_RATE
+#if (1) == DISP_AI_FRAME_RATE
         int arraySum = std::accumulate(array_drp_time, array_drp_time + SIZE_OF_ARRAY(array_drp_time), 0);
         double arrayAvg = 1.0 * arraySum / SIZE_OF_ARRAY(array_drp_time);
         ai_fps = 1.0 / arrayAvg * 1000.0 + 0.5;
