diff --git a/python/setup.py b/python/setup.py
index 67cdbf4..86bbc15 100644
--- a/setup.py
+++ b/setup_n.py
@@ -69,7 +69,6 @@ setuptools.setup(
     cmdclass=cmdclass,
     python_requires='>=3',
     zip_safe=False, # Since we load so file from the filesystem, we can not run from zip file
-    setup_requires=['cffi>=1.0', 'requests', 'tqdm', 'srt', 'websockets'],
     install_requires=['cffi>=1.0', 'requests', 'tqdm', 'srt', 'websockets'],
     cffi_modules=['vosk_builder.py:ffibuilder'],
 )
